src/Fantomas.Tests/ElmishTests.fs

```diff
@@ -433,10 +433,9 @@ table [ ClassName "table table-striped table-hover mb-0" ] [
         tokenDetailRow "ColorClass" (str colorClass)
         tokenDetailRow "CharClass" (str charClass)
         tokenDetailRow "Tag" (ofInt tag)
-        tokenDetailRow "FullMatchedLength"
-            (span [ ClassName "has-text-weight-semibold" ] [
-                ofInt fullMatchedLength
-             ])
+        tokenDetailRow "FullMatchedLength" (span [ ClassName "has-text-weight-semibold" ] [
```

Comment:
This seems a bit weird as the span block is multiline.
So I would expect it to be more like
```fsharp
tokenDetailRow
    "FullMatchedLength"
    (span [ ...

    ])
```

Reply:
I believe the problem here is the following:
When the long expression is checked the printer tries to put all parameters in one line. Further down in the recursive calls where the array is printed it is detected that the page width will be exceeded if the array is printed in one line so it is split.
That satisfies the maximum width for the long expression again.

Reply:
Might need your help on what's the best way to fix that one.

Reply:
The code now makes
```fsharp
        tokenDetailRow
            "FullMatchedLength"
            (span [ ClassName "has-text-weight-semibold" ] [
                ofInt fullMatchedLength
             ])
```
of it. This is correct behaviour in my opinion. The function call doesn't fit on the remainders of the line because of the arguments doesn't fit. This makes sense.
I think the exception here will be lambda's, if you have those, you might want to keep the original behaviour. See next comment.